runtime/queries/just/highlights.scm

```diff
@@ -0,0 +1,35 @@
+(assignment (NAME) @variable)
+(alias (NAME) @variable)
+(value (NAME) @variable)
+(parameter (NAME) @variable)
+(setting (NAME) @keyword)
+(setting "shell" @keyword)
+
+(call (NAME) @function)
+(dependency (NAME) @function)
+(depcall (NAME) @function)
+(recipeheader (NAME) @function)
+
+(depcall (expression) @parameter)
+(parameter) @parameter
+(variadic_parameters) @parameter
+
+["if" "else"] @conditional
+
+(string) @string
+
+(boolean ["true" "false"]) @boolean
+
+(comment) @comment
+
+; (interpolation) @string
+
+(shebang interpreter:(TEXT) @keyword ) @comment
+
+["export" "alias" "set"] @keyword
+
+["@" "==" "!=" "+" ":="] @operator
+
+[ "(" ")" "[" "]" "{{" "}}" "{" "}"] @punctuation.bracket
+
+(ERROR) @error
```

Comment:
Eventually we want to add error node highlighting with a config option. It's very noisy when writing new code for most parsers so we've been removing the highlight in general

```suggestion
```